src/pages/UserSite/HomePage/homePage.tsx

```diff
@@ -9,6 +9,8 @@ import Reviews from './../../../interfaces/Reviews';
 import RecommendComp from './../../../components/UserSite/HomePage/RecommendComp';
 import ReviewsComp from './../../../components/UserSite/HomePage/ReviewsComp';
 
+// test
+
 export default class HomePage extends React.Component{
```

Comment:
Staten din har ikke types og kommer opp som type never. Props har type any, og any bør unngås der du kan.
Du kan sette types på props og state sånn ```class HomePage extends React.Component<PROPS, STATE> { }```. Jeg hadde nok laget en interface for staten siden satt et tomt object for props.